HedgeModManager/Languages/ja-JP.xaml

```diff
@@ -0,0 +1,241 @@
+    <system:String x:Key="StatusUICheckingForLoaderUpdate">{0} のアップデートを確認中</system:String>
+    <system:String x:Key="StatusUILoaderUpToDate"         >{0} は最新です</system:String>
+    <system:String x:Key="StatusUIFailedLoaderUpdateCheck">{0} のアップデートの確認に失敗</system:String>
+    <system:String x:Key="StatusUIInstalledLoader"        >{0} をインストールしました</system:String>
+    <system:String x:Key="StatusUIDeletedMod"             >{0} を削除しました</system:String>
+    <system:String x:Key="StatusUIModsDBSaved"            >MODデータベースを保存しました</system:String>
+    <system:String x:Key="StatusUIModsDBLocationChanged"  >データベースの場所を変更しました</system:String>
+    <system:String x:Key="StatusUIGameChange"             >{0} に変更しました</system:String>
+    <system:String x:Key="StatusUIDownloadingCodes"       >{0} のコードをダウンロード中</system:String>
+    <system:String x:Key="StatusUIDownloadFinished"       >ダウンロード完了</system:String>
+    <system:String x:Key="StatusUIDownloadFailed"         >ダウンロード失敗</system:String>
+    <system:String x:Key="StatusUICodeUpdatesAvailable"   >コードのアップデートが利用可能です!</system:String>
+    <system:String x:Key="StatusUINoCodeUpdatesFound"     >コードのアップデートはありません</system:String>
+
+    <!-- Dialog UI Strings -->
+    <system:String xml:space="preserve" x:Key="DialogUINoGameDirAccess">Hedge Mod Managerはゲームフォルダにアクセスできませんでした。&#x0a;以下のフォルダとフォルダ内の全てのファイルについて、Usersグループが&quot;書き込み&quot;を許可する設定になっていることを確認してください。&#x0a;&#x0a;{0}</system:String>
+    <system:String xml:space="preserve" x:Key="DialogUIDeleteMod"      >本当に &quot;{0}&quot; を削除しますか?&#x0a;この操作は元に戻せません!</system:String>
+    <system:String xml:space="preserve" x:Key="DialogUIModNewest"      >{0} は既に最新です。</system:String>
+    <system:String xml:space="preserve" x:Key="DialogUIModUpdate"      >新しいバージョンの {0} が利用可能です! ({1})</system:String>
+    <system:String xml:space="preserve" x:Key="DialogUIModCancelUpdate">本当にアップデートをキャンセルしますか?&#x0a;{0} が破損する可能性があります。</system:String>
+    <system:String xml:space="preserve" x:Key="DialogUINoDecompressor" >7-Zip または WinRARが見つかりませんでした。&#x0a;これはMODデータの解凍に必要です。&#x0a;&#x0a;どちらかがシステムにインストールされていることを確認してください。</system:String>
+
+    <!-- Status UI Lang Stuff Strings -->
+    <system:String x:Key="StatusUIUpdateCompletedSingular"      >完了</system:String>
+    <system:String x:Key="StatusUIUpdateCompletedPlural"        >完了</system:String>
+    <system:String x:Key="StatusUIUpdateFailedSingular"         >失敗</system:String>
+    <system:String x:Key="StatusUIUpdateFailedPlural"           >失敗</system:String>
+
+    <!-- Mod Config UI Strings -->
+    <system:String x:Key="ModConfigUITitle">{0} の設定</system:String>
+
+    <!-- Mod Description UI Strings -->
+    <system:String x:Key="ModDescriptionUIAbout">{0} について</system:String>
+    <system:String x:Key="ModDescriptionUIMadeBy">Made by</system:String>
+    <system:String x:Key="ModDescriptionUIMadeOn">Made on</system:String>
+    <system:String x:Key="ModDescriptionUIDate">on</system:String>
```

Comment:
These are not translated. While it is formatted to work in English, can you translate this so that it looks correct in the UI? Maybe in the future we can get this changed to a format string.

Reply:
It is difficult to properly translate a sentence like `A made by B on C` without a format string because Japanese grammar differs from English.  
However, words like `made by' and `made on' are widely known in Japan, and it would not be a problem if they were not translated.

Reply:
Ok, then we can leave it as is for now.